### PR TITLE
Feature/offpay 752

### PR DESCRIPTION
**Pull request by brian-digital**

### CODE REVIEW

File: app/uk/gov/hmrc/offpayroll/views/interview/printResult.scala.html [hunk 1]
```diff
@@ -128,9 +128,26 @@
...
+        /*background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACMAAAAjCAQAAAC00HvSAAABZklEQVR4Aa3WsWoTUBTG8V9v4yQElywRAlYzJVCQdMrgmKVDce3m3mfI0ofoY7QQfAOngmQIOGgzOGRpQYhLUynHZjb3JtL7/+8f53DhfmfPZlqGjvR1tTUtLXw3c+2LWzsycG4qNjp1bmArHWM3ouiNsY4CIxOxkxMjGU7NxM7OnG4OmYv/cv5v0KgwSWmiEdAAHWd6NvLeHsJXG+g5881PgLHI+RBrHkLOMcCg9MS/Y82y9PwDEk4cyHIPVrIcOCFpOVZgtS2GY61k6PCZMYeGyRFsW+pekaOkzzOnoZ90K8R0k3aFpdpJs8I0zWRZIWaZLCosteBKlNyPF0/uh5JXyUyRR3+efFRk1nCtyBufrFy4U+CalqnI+Tp+xZof8TLknGoltyayfPQKvPVBlonbhEtzGe4A8kvNXW79/RrxOdZc7PD70cn301704l2przoVm6FCT1VozQodXuuiqH/fVLq2/gLNMwwtCpJpLgAAAABJRU5ErkJggg==);*/
```
Comment (peteanning) [inline app/uk/gov/hmrc/offpayroll/views/interview/printResult.scala.html]: You need to merge the develop branch into this feature before commiting

File: app/uk/gov/hmrc/offpayroll/controllers/PrintController.scala [hunk 2]
```diff
@@ -67,8 +67,8 @@ class PrintController @Inject() (pdfGeneratorConnector: PdfGeneratorConnector) e
         val interviewAsRawList = compressedInterview.asRawList
         val fragments = fragmentService.getAllFragmentsForInterview(compressedInterview.asMap) ++ fragmentService.getFragmentsByFilenamePrefix("result")
         val resultPageHelper = ResultPageHelper(interviewAsRawList, decisionType, fragments, cluster, esi)
-
-        Future.successful(BadRequest(uk.gov.hmrc.offpayroll.views.html.interview.display_decision(decision, interviewAsRawList, esi, compressedInterviewRawString, resultPageHelper, error)))
+//        val renderShowHideToggle = false
```
Comment (peteanning) [inline app/uk/gov/hmrc/offpayroll/controllers/PrintController.scala]: No commented out code please

File: app/uk/gov/hmrc/offpayroll/controllers/PrintController.scala [hunk 3]
```diff
@@ -110,7 +110,8 @@ class PrintController @Inject() (pdfGeneratorConnector: PdfGeneratorConnector) e
         val decision = Decision(session,getDecisionType(printResult.decisionResult),printResult.decisionVersion,printResult.decisionCluster)
         val resultPageHelper = ResultPageHelper(interview, getDecisionType(printResult.decisionResult), fragments, printResult.decisionCluster, printResult.esi)
 
-        val body: Html = uk.gov.hmrc.offpayroll.views.html.interview.printResult(printResult, decision, resultPageHelper, emptyForm)
+//        val renderShowHideToggle = false
```
Comment (peteanning) [inline app/uk/gov/hmrc/offpayroll/controllers/PrintController.scala]: ditto commented code

File: app/uk/gov/hmrc/offpayroll/views/interview/printResult.scala.html (hunk 1 above)
Comment (brian-digital) [inline app/uk/gov/hmrc/offpayroll/views/interview/printResult.scala.html]: will do

File: app/uk/gov/hmrc/offpayroll/controllers/PrintController.scala (hunk 3 above)
Comment (brian-digital) [inline app/uk/gov/hmrc/offpayroll/controllers/PrintController.scala]: oops

**Pull request closed without merging**